src/Sylius/Component/Channel/Model/Channel.php

```diff
@@ -64,7 +64,7 @@ class Channel implements ChannelInterface
     protected $updatedAt;
 
     /**
-     * Constructor
+     * Constructor.
```

Comment:
Yes but it's for passing the <code>missing doc block</code> rule of php code sniffer